springboard_iogt/templates/base.jinja2

```diff
@@ -39,14 +40,13 @@
 
     <div id="submenu">
         {% block menu %}
-        <div class="h1">{% trans %}Explore IoGT{% endtrans %}</div>
-        <ul>
+        <div class="h1"><div class="detail">{% trans %}Explore IoGT{% endtrans %}</div></div>
+        <ul class="options">
         {% for section in content_sections %}
             {% set section_url=content_section_url_overrides.get(section.slug, 'content_section'|route_url(slug=section.slug)) %}
-            <li><a href="{{ section_url }}">{{ section.title }}</a>
+            <li class="{{section.slug}}"><a href="{{ section_url }}">{{ section.title }}<p>{{section.descriptor}}</p></a>
             </li>
         {% endfor %}
-        <li><a href="{{ 'home'|route_url }}">{{ _("Home") }}</a></li>
```

Comment:
We decided to keep home at the bottom of the menu, even though it's not on the designs. Sorry for the confusion